src/yuzu/startup_checks.cpp

```diff
@@ -150,10 +169,29 @@ bool SpawnChild(const char* arg0, PROCESS_INFORMATION* pi, int flags) {
                                                 pi             // lpProcessInformation
     );
     if (!process_created) {
-        std::fprintf(stderr, "CreateProcessA failed with error %lu\n", GetLastError());
+        fmt::print(stderr, "CreateProcessA failed with error {}\n", GetLastError());
         return false;
     }
 
     return true;
 }
+#elif defined(YUZU_UNIX)
+pid_t SpawnChild(const char* arg0) {
+    const pid_t pid = fork();
+
+    if (pid == -1) {
+        // error
+        const int err = errno;
+        fmt::print(stderr, "fork failed with error {}\n", err);
+        return pid;
+    } else if (pid == 0) {
+        // child
+        execl(arg0, arg0, nullptr);
+        const int err = errno;
+        fmt::print(stderr, "execl failed with error {}\n", err);
+        _exit(0);
+    }
+
+    return pid;
```

Comment:
What about using [`posix_spawn()`](https://pubs.opengroup.org/onlinepubs/9699919799/functions/posix_spawn.html)?

```suggestion
    pid_t pid;
    char* const arg0v[] = { const_cast<char*>(arg0), NULL };

    const int err = posix_spawn(&pid, arg0, NULL, NULL, arg0v, NULL);
    if (err != 0) {
        fmt::print(stderr, "posix_spawn failed with error {}\n", err);
        return -1;
    }

    return pid;
```